### ISSUE TITLE
Allow Reset-EpiDeployment when status is not AwaitingVerification

### ISSUE THREAD

**Issue by erdihu** [Epinova/epinova-dxp-deployment#147]
I do not know if this limitation is coming from EpiCloud or Epinova package but I wanted to ask here first in case the limitation exists only at this package.

Sometimes the slot environment crashes directly after the deployment warmup phase gets stuck until it times out. It would be nice to reset the deployment slot manually in such cases. Currently the current message is given if reset is run:

`Environment Integration is not in status AwaitingVerification. We do not need to reset this environment.`

**Comment by ovelartelius** [comment]
Hi @erdihu. Sorry for late answer. That limitation is from the EpiCloud, or actually from the DXP/Azure environment. There is no chans to cancel when you are in the middle of a operation. But it could be other workarounds that you could use.
Is it possible to give me some more detailed information about how your pipeline is setup so that I can understand a little bit more about your situation?
Regards Ove

**Comment by erdihu** [comment]
Hi @ovelartelius, no problem at all. Thank you for this package, it works great!

Sometimes the warmup process takes very long but after couple of minutes the process has started it is still possible to surf into the slot environment. In case there are any errors on the site, we have to wait until the warmup finishes in order to cancel the deployment. I was wondering if it is possible to just cancel but if that is not allowed I don't think there's anything else to do than to wait until completion. 

Usually I just use web deploy for the integration but recently I had to test something in the slot environment and for those who get used to the speed of the web deploy, the time needed for the slot deployment feels like forever.

**Comment by ovelartelius** [comment]
Hi @erdihu i agree that the deploy time is very high. Everybody struggle with that problem. But are you using direct deploy? That will make your deployment time much faster. https://www.epinova.se/nyheter-och-blogg/utvecklarbloggen/2021/directdeploy-with-epinova-dxp-deployment/
Another thing that we did to make the deploy faster was actually to spend some time making sure that the website started fast and that we did not load any unnecessarily when starting the website. Like to many routes, unnecessarily init jobs etc. That made our site in that project  been able to warmup very much faster. And that was positive in deploy context also.
I hope that it will help.
